collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -37,6 +38,19 @@
         ]
     }
 }
+_TPS_REGEX = re.compile(
+    '^.*: .*?'                     # Message lead-in
+    '([0-9]{1,2}.[0-9]{1,2}), .*?' # 1-minute TPS value
```

Comment:
if format is `1.11` and not `01.11` should be `([1-9]{1,2}.[0-9]{1,2})`

and if you unsure about decimal part please do [0-9]+